src/main/java/com/aca/homework/week15/sql/generate/UserCrudRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.aca.homework.week15.sql.generate;
+
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class UserCrudRepository {
+
+    private final Connection connection;
+
+    public UserCrudRepository(Connection connection) {
+        this.connection = connection;
+    }
+
+    public User save(User user) {
+
+        try {
+            connection.createStatement().execute(new SQLGenerator().generateInsert(user));
```

Comment:
please do not hard wire the SQLGenerator creation. instead, require it in constructor